File: webflux-dialogue/src/main/java/com/study/webflux/rag/domain/dialogue/model/CompletionResponse.java
package com.study.webflux.rag.domain.dialogue.model;

public record CompletionResponse(
	String content,
	String model,
	int tokensUsed
) {
	public static CompletionResponse of(String content) {
		return new CompletionResponse(content, null, 0);
	}
}

File: app-miyou/src/main/kotlin/com/miyou/app/domain/dialogue/model/CompletionResponse.kt
package com.miyou.app.domain.dialogue.model

data class CompletionResponse(
    val content: String,
    val model: String?,
    val tokensUsed: Int,
) {
    companion object {
        fun of(content: String): CompletionResponse = CompletionResponse(content, null, 0)
    }
}
